app/src/test/java/com/kickstarter/viewmodels/ActivityFeedViewModelTest.java

```diff
@@ -59,7 +59,9 @@ public void testClickingInterfaceElements() {
     goToLogin.assertNoValues();
     goToProject.assertNoValues();
     goToProjectUpdate.assertNoValues();
+    koalaTest.assertValues("Activity View");
```

Comment:
It might be nice to pull out these strings into one place rather than duplicating them across the codebase